stingray/events.py

```diff
@@ -719,50 +666,6 @@ def filter_energy_range(self, energy_range, inplace=False, use_pi=False):
 
         return self.apply_mask(mask, inplace=inplace)
 
-    def apply_mask(self, mask, inplace=False):
```

Comment:
So I'm assuming that this now inherits from the superclass? I'm wondering if it might still be worth having a custom docstring here for eventless specifically?

Reply:
I don't know, it does exactly the same as `StingrayTimeseries`

Reply:
Eh, it's a minor thing. I was wondering whether it confuses people if they look at a docstring for an EventList object and the docstring just talks about TimeSeries objects everywhere :) But maybe I'm not giving our users enough credit here. :D